application/src/test/java/uk/nhs/hee/tis/revalidation/service/RecommendationServiceTest.java

```diff
@@ -788,6 +794,36 @@ void shouldReturnLatestRecommendationsList() {
     assertThat(traineeRecommendationRecordDto.getGmcNumber(), is(nullValue()));
   }
 
+  @Test
+  void shouldCheckDraftRecommendations() {
+    final var gmcNumber2 = faker.number().digits(7);
```

Comment:
Can't see any assertions for the data `gmcNumber2`, so I think this is not necessary.

Reply:
Used in the assertion now